app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -1363,6 +1363,21 @@ public boolean isOverrideEnvPathEnabled() {
         return SettingsStore.getInstance(this).isEnvironmentOverrideEnabled();
     }
 
+    @Keep
+    @SuppressWarnings("unused")
+    public void checkTogglePassthrough() {
+        switch (DeviceType.getType()) {
+            case DeviceType.LenovoA3:
+                SettingsStore.START_WITH_PASSTHROUGH_DEFAULT = true;
```

Comment:
Hmm two things here:
* it's ok to have a simple if as there are no more devices requiring passthrough by default
* The settingsstore values should be final and not editable. You should add the logic for the A3 when this DEFAULT is used instead of editing it. Another option is unconditionally calling togglePassthrough bellow for the A3 if the setting is false.

Reply:
Let's stop using the `START_WITH_PASSTHROUGH_DEFAULT` settings store value but rather use the static method to determine the default value.